src/test/java/cloud/fogbow/ras/core/plugins/interoperability/aws/attachment/v2/AwsV2AttachmentPluginTest.java

```diff
@@ -271,10 +270,10 @@ public void testMountAttachmentInstanceWithoutVolumes() throws InstanceNotFoundE
 	}
 	
 	private AttachmentInstance createAttachmentInstance() {
-        String id = FAKE_VOLUME_ID;
+        String id = testUtils.FAKE_VOLUME_ID;
         String cloudState = AwsV2StateMapper.ATTACHED_STATE;
-        String computeId = FAKE_INSTANCE_ID;
-        String volumeId = FAKE_VOLUME_ID;
+        String computeId = testUtils.FAKE_INSTANCE_ID;
+        String volumeId = testUtils.FAKE_VOLUME_ID;
```

Comment:
Use the TestUtils class constants statically.

Correct the other occurrences in this class and in all other test classes.